liquid_node/commands.py

```diff
@@ -137,6 +138,25 @@ def get_failed_checks():
     raise RuntimeError(msg)
 
 
+def deploystats():
+    """Get memory and CPU usage for the deployment"""
+
+    def get_all_res():
+        jobs = (nomad.parse(get_job(path)) for _, path in config.jobs)
+        for spec in jobs:
+            yield from nomad.get_resources(spec)
+
+    total = collections.defaultdict(int)
+    for _, _type, res in get_all_res():
+        for key in ['MemoryMB', 'CPU', 'DiskMB']:
+            totalkey = _type + " " + key
```

Comment:
how 'bout `f'{_type} {key}'` and merge with the next line? It's even more readable that way.